### DISCUSSION TITLE
Is there a way to stream the results?

### DISCUSSION THREAD

**Discussion by resticker** [website-scraper/node-website-scraper#502] [Q&A] [answered]
Bit of a noob with node, but would like to have my server stream the results of the website scraper into a archiver, so I can download all the files at once as a zip file

Is it possible to accomplish this? I only see methods for saving to a local drive

Thanks!

**Comment by aivus** [comment, marked as answer]
Hey @resticker 

You can take a look on https://github.com/website-scraper/demo and set it up for yourself, but it's quite outdated.

**Reply by resticker** [reply]
Thank you!
